Assets/MixedRealityToolkit/Utilities/Editor/ScreenshotUtility.cs

```diff
@@ -0,0 +1,152 @@
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Utilities/Take Screenshot/4x Resolution")]
+        private static void CaptureScreenshot4x()
+        {
+            CaptureScreenshot(GetScreenshotFileName(), 4);
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Utilities/Take Screenshot/4x Resolution (Transparent Background)")]
+        private static void CaptureScreenshot4xAlphaComposite()
+        {
+            CaptureScreenshot(GetScreenshotFileName(), 4, true);
+        }
+
+        /// <summary>
+        /// Captures a screenshot with the current main camera's clear color.
+        /// </summary>
+        /// <param name="path">The path to save the screenshot to.</param>
+        /// <param name="superSize">The multiplication factor to apply to the native resolution.</param>
+        /// <param name="transparentClearColor">True if the captured screenshot should have a transparent clear color. Which can be used for screenshot overlays.</param>
+        /// <param name="camera">The optional camera to take the screenshot from.</param>
+        /// <returns>True on successful screenshot capture, false otherwise.</returns>
+        public static bool CaptureScreenshot(string path, int superSize = 1, bool transparentClearColor = false, Camera camera = null)
+        {
+            if (string.IsNullOrEmpty(path) || superSize <= 0)
+            {
+                return false;
+            }
+
+            // If a transparent clear color isn't needed and we are capturing from the default camera, use Unity's screenshot API.
+            if (!transparentClearColor && (camera == null || camera == Camera.main))
+            {
+                ScreenCapture.CaptureScreenshot(path, superSize);
+
+                Debug.LogFormat("Screenshot captured to: {0}", path);
+
+                return true;
+            }
+
+            // Make sure we have a valid camera to render from.
+            if (camera == null)
+            {
+                camera = Camera.main;
+
+                if (camera == null)
+                {
+                    Debug.Log("Failed to acquire a valid camera to capture a screenshot from.");
+
+                    return false;
+                }
+            }
+
+            // Create a camera clone with a transparent clear color.
+            var renderCamera = new GameObject().AddComponent<Camera>();
+            renderCamera.transform.position = camera.transform.position;
+            renderCamera.transform.rotation = camera.transform.rotation;
+            renderCamera.clearFlags = transparentClearColor ? CameraClearFlags.Color : camera.clearFlags;
+            renderCamera.backgroundColor = transparentClearColor ? new Color(0.0f, 0.0f, 0.0f, 0.0f) : camera.backgroundColor;
+            renderCamera.fieldOfView = camera.fieldOfView;
+            renderCamera.nearClipPlane = camera.nearClipPlane;
+            renderCamera.farClipPlane = camera.farClipPlane;
```

Comment:
Would be nice to capture orthographic cameras as well.

Reply:
That's a good idea, let me try that out.